fastlane/Fastfile

```diff
@@ -760,66 +761,56 @@ REPOSITORY_NAME = 'WordPress-Android'
       github_release_prefix: "",
       exclusions: []
     },
-  ]
+  ].freeze
 
-  private_lane :send_strings_for_translation do |options|
-    sh("cd .. && mkdir -p #{update_strings_path} && cp #{main_strings_path} #{update_strings_path} && git add #{update_strings_path}strings.xml")
-    sh('git diff-index --quiet HEAD || git commit -m "Send strings to translation."')
-    sh('git push origin HEAD')
-  end
-
-  private_lane :commit_strings do |options|
-    if options[:auto_commit] then
-       sh("cd .. && git add #{main_strings_path}")
-       sh("git commit -m 'Update strings for translation'")
-       sh('git push origin HEAD')
-    else
-      UI.important("Your #{main_strings_path} has changed.")
-      UI.input('Please, review the changes, commit them and press return to continue.')
+  lane :update_frozen_strings_for_translation do |options|
+    Dir.chdir('..') do
+      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
+      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
     end
+    git_commit(path: File.join(FROZEN_STRINGS_DIR_PATH, 'strings.xml'), message: 'Freeze strings for translation', allow_nothing_to_commit: true)
```

Comment:
> replace it with a direct call to the built-in git_commit fastlane action (with a clearer git message as well)

Both code and message are clearer indeed 👍

Reply:
Nitpick / Future improvement suggestion.

I like adding the version number to this kind of commits every time I can. The reason being that in a filtered `git log` output

```
Freeze strings for translation 1.5
Freeze strings for translation 1.4
Freeze strings for translation 1.3
Freeze strings for translation 1.2
Freeze strings for translation 1.1
```

is more clear than

```
Freeze strings for translation
Freeze strings for translation
Freeze strings for translation
Freeze strings for translation
Freeze strings for translation
```

Reply:
> So even if I like the idea, instead of spending too much time of that detail for now, I figured we'd keep that message at least for now, and we could always revisit in a later PR 🙂

Sounds good to me! Thank you for trying.